azure-mgmt-cdn/src/main/java/com/microsoft/azure/management/cdn/ProfileResourceState.java

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
in the future, pls do not include auto-gen'd code with hand-written code in the same PR. Makes full code reviews pretty much impossible :-)